contracts/src/v0.8/ccip/interfaces/IMessageTransformer.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {Internal} from "../libraries/Internal.sol";
+
+/// @notice Interface for plug-in message hook contracts that transform OffRamp & OnRamp messages.
+///         The transformer functions are expected to revert on transform failures.
```

Comment:
nit: start the comment at the start of the line. 
nit: what does `plug-in` mean here? I think the comment would be better without it

Reply:
makes sense. removed `plug-in`